launcher/minecraft/auth/steps/MinecraftProfileStep.cpp

```diff
@@ -44,7 +44,7 @@ void MinecraftProfileStep::onRequestDone(
     qCDebug(authCredentials()) << data;
     if (error == QNetworkReply::ContentNotFoundError) {
         // NOTE: Succeed even if we do not have a profile. This is a valid account state.
-        if(m_data->type == AccountType::Mojang) {
+        if(m_data->type == AccountType::Mojang || m_data->type == AccountType::CustomYggdrasil) {
```

Comment:
I agree, I've changed it to grant `canPlayMinecraft` and `ownsMinecraft` when the account is created, like we do for offline accounts, instead of doing it here.